src/libraries/PSMLib.sol

```diff
@@ -128,6 +128,8 @@ library PSMLib {
     /*** Helper functions                                                                       ***/
     /**********************************************************************************************/
 
+    // NOTE: As we are swapping USDC to USDS and vice versa, we are not using 
```

Comment:
`    // NOTE: As swaps are only done between USDC and USDS and vice versa, using `_forceApprove` is unnecessary`

Reply:
fixed